seqr/views/apis/staff_api_tests.py

```diff
@@ -648,8 +648,8 @@ def test_anvil_export(self, mock_export_multiple_files):
         _check_login(self, url)
 
         # Create a SavedVariant row needed by this test
-        saved_variant = SavedVariant.objects.get(guid = "SV0000001_2103343353_r0390_100")
-        variant_tag_type = VariantTagType.objects.get(name = "Known gene for phenotype")
+        saved_variant = SavedVariant.objects.get(guid__exact = "SV0000001_2103343353_r0390_100")
```

Comment:
You don't need to use `__exact`, by default if you just do `guid=...` it will do an exact lookup and its easier to read that way

Reply:
Removed. I also removed the `__exact` for `name` because it makes no different in the test.